packages/@tailwindcss-upgrade/src/template/prepareConfig.ts

```diff
@@ -0,0 +1,101 @@
+import { __unstable__loadDesignSystem, compile } from '@tailwindcss/node'
```

Comment:
Can we rename this whole file to `prepare-config.ts` instead of `prepareConfig.ts`?